concourse/pipeline.yml

```diff
@@ -213,7 +213,7 @@ jobs:
           CF_APP_NAME: govuk-account-manager
           CF_SPACE: production
           CF_STARTUP_TIMEOUT: 15 # minutes
-          APP_INSTANCES: 20
+          CF_APP_SCALING_POLICY: policy.json
```

Comment:
I've not tried passing a file path before... 
i'm not entrely sure if this will work... or if concourse will be `pwd`-d into the /concourse directory if so...
I might need more of a path?

Other option is perhaps to set up a separate concourse task kind of like "update-pipeline" that just monitors the relevant policy folder on github and then does autoscaling policy applicaiton outside of the nomal deploy pipeline?

Would be keen for thoughts

Reply:
The deploy task is run in the git directory, so this is fine.

Reply:
Also, we could have a separate scaling task, but the deploy task will pick up changes to it anyway